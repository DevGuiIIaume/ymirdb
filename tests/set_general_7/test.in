set a 1
set b a 2
set c 3
set b c a 4
forward a
forward b
forward c
backward a
backward b
backward c
bye
