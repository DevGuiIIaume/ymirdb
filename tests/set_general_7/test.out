> ok

> ok

> ok

> ok

> nil

> a, c

> nil

> b

> nil

> b

> bye
